setup.cfg

```
@@ -18,6 +18,7 @@ combine_as_imports = True
 addopts = -rxXs
 markers =
   copied_from(source, changes=None): mark test as copied from somewhere else, along with a description of changes made to accodomate e.g. our test setup
+  network: marks tests which require network connection
```

Should we embed a bit of the rationale for introducing this marker?

```suggestion
  network: marks tests which require network connection. Allows skipping such tests, e.g. in build environments with network disabled.
```